src/hooks/useInterval.js

```diff
@@ -0,0 +1,26 @@
+import React, {useEffect, useRef } from 'react'
```

Comment:
Codacy found an issue: ['React' is defined but never used.](https://app.codacy.com/gh/treetoplearning/heartwood/pullRequest?prid=5837031)